doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -15,6 +15,7 @@
 * [Ordering between:$
  different attribute value types](#ordering-between-different-attribute-value-types)
 * [Initial notifications](#initial_notifications)
+* [Oneshot Subscription](#oneshot_subscriptions)
```

Comment:
I have been testing and the rigth one is `-`. Otherwise, the link doesn't work. I was confused by the above line (`#initial_notifications`) which is previous to the PR but incorrect :(

Could you change it back to `-`, please? At the same time, could you also fix `#initial_notifications` so it be `#initial-notifications`?

Sorry for the mess. My fault. Thanks!

Reply:
fixed in 06f6526